nixos/modules/system/boot/stage-1.nix

```diff
@@ -454,7 +454,7 @@ in
       {
         symlink = "/etc/modules-load.d/nixos.conf";
         object = pkgs.writeText "nixos.conf"
-          (lib.concatStringsSep "\n" (config.boot.initrd.availableKernelModules ++ config.boot.initrd.kernelModules));
+          (lib.concatStringsSep "\n" config.boot.initrd.kernelModules);
```

Comment:
Can you add some rational to "only load the minimum in modules-load.d"? Perhaps we did it over excessive? Would be good to add to the commit message in case this backfires.

Reply:
No, including `availableKernelModules` was a mistake in my code in the first place. NixOS has never auto-loaded those; they're there for the kernel to request to be loaded if needed.